Gemfile

```diff
@@ -54,10 +54,10 @@ gem '3scale_time_range', '0.0.6'
 gem 'statsd-ruby', require: false
 
 # Sidekiq
-gem 'sidekiq', '~> 6.4.0', require: %w[sidekiq sidekiq/web]
-gem 'sidekiq-batch'
+gem 'sidekiq', '~> 7', require: %w[sidekiq sidekiq/web]
+gem 'sidekiq-batch', github: '3scale/sidekiq-batch', branch: 'redis-client'
 gem 'sidekiq-cron', require: %w[sidekiq/cron sidekiq/cron/web]
-gem 'sidekiq-throttled'
+gem 'sidekiq-throttled', '~> 1.4.0'
```

Comment:
Does this support ActiveJob fully now?

Reply:
Yes, at least all scenarios I found locally. I wouldn't be surprised if we find new failing scenarios in staging or production.